### ISSUE TITLE
Json POST requests don't get sent to the server (iOS)

### ISSUE THREAD

**Issue by shaisa** [ufna/VaRest#147]
I'm using 4.18.3 for a mobile app for iOS.
I want to POST a Json request to a Google App Engine server that I run, and get a reply from the server to the app.
Using command line from my Mac (curl) to communicate with the server works fine.

When using the same POST command and Json request on iOS, it receives a reply from the server only after I delete the app and install.
- If I reinstall the app, the requests are not sent right and/or the replies are not returned / visible
- If some time passes, same issue.

I've checked the server side - it seems like the requests are not reaching the server...

I'm running on iOS so debugging packets is a problem...
Is it a familiar problem? Suggestions for fix?

**Comment by ufna** [comment]
Hi @shaisa

Try to turn the VaRest and Http modules log level to VeryVerbose and then launch your app with logs enabled. Collect it with XCode Device manager and check what happens with requests.

**Comment by shaisa** [comment]
@ufna thanks - if something interesting comes up I'll open a new issue.
